packages/extension-api/src/extension-api.d.ts

```diff
@@ -1042,6 +1042,18 @@ declare module '@podman-desktop/api' {
      * Whether or not to emit an OS notification noise when showing the notification.
      */
     silent?: boolean;
+    /**
+     * The type of the notification. Default value: info
+     */
+    type?: NotificationType;
+    /**
+     * displayed below the description, centered in the notification card. It contains actions (like markdown commands/buttons and links)
+     */
+    actions?: string;
+    /**
+     * the notification will be visible in the dashboard
```

Comment:
```suggestion
     * the notification will be visible in the dashboard
```

I think we might not say it'll go in the dashboard, it's how it's implemented now but it could change
We might say it in a generic way: this notification will be highlighted to the user so it draws attention.